src/main/java/gregtech/api/worldgen/config/WorldGenRegistry.java

```diff
@@ -172,11 +175,35 @@ public void reinitializeRegisteredVeins() throws IOException {
                 deposit.initializeFromConfig(element);
                 // Adds the registered definition to the list of all registered definitions
                 registeredDefinitions.add(deposit);
+                definitionMap.put(depositName, deposit);
             } catch (RuntimeException exception) {
                 GTLog.logger.error("Failed to parse worldgen definition {} on path {}", depositName, worldgenDefinition, exception);
             }
         }
+
+        for(OreDepositDefinition definition : addonRegisteredDefinitions) {
+
+            JsonObject element = FileUtility.tryExtractFromFile(worldgenRootPath.resolve(definition.getDepositName()));
+
+            if(element == null) {
+                GTLog.logger.error("Addon mod tried to register bad ore definition at {}", definition.getDepositName());
+                addonRegisteredDefinitions.remove(definition);
+                break; //break or continue?
```

Comment:
I'd say this should be a `continue`. Attempting to load and failing on all bad veins will be more useful, since it'd catch all possible broken veins at once when debugging. Also, it will allow a single bad vein to not cause every subsequent vein to not load, which will mean that players' world-gen will not be entirely blocked off by a single bad mod.